### DISCUSSION TITLE
oh-my-posh on pwsh (macOS) stopped working after Ventura upgrade

### DISCUSSION THREAD

**Discussion by Tuttu** [JanDeDobbeleer/oh-my-posh#3837] [Troubleshoot] [answered]
Hello!

```
OMP version: 16.4.0
bash version: GNU bash, version 3.2.57(1)-release (arm64-apple-darwin22)
pwsh version: 7.3.3
```

It looks like the macOS update to Ventura broke oh-my-posh for me on pwsh.
When I first setup OMP, I followed the guide and got it working using my custom config file. It's started using this line in my PowerShell profile: 
`oh-my-posh init pwsh --config "~/mdb.omp.json" | Invoke-Expression`
However, it looks like it stopped working after an upgrade of my OS to macOS Ventura. Now, I get this error instead: 
```
oh-my-posh: The term 'oh-my-posh' is not recognized as a name of a cmdlet, function, script file, or executable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
```
I have updated OMP to the latest version using Brew but it doesn't fix the issue. On the other side, OMP is working perfectly when used in Bash. As I'm still quite new using macOS, I have not a single idea what could cause this difference in behavior between Bash and pwsh. Any idea?

**Comment by JanDeDobbeleer** [comment, marked as answer]
@Tuttu yes, you probably don't have the homebrew bin folder added to your PATH when using pwsh.

**Reply by Tuttu** [reply]
Looks like it must be that indeed. Now that you mention them, I have looked at the `PATH` variable from zsh and from pwsh and the Brew folders (`/opt/homebrew/bin:/opt/homebrew/sbin`) are missing in pwsh. 
I will add them and test again. :)

**Reply by Tuttu** [reply]
I can confirm that it's indeed the issue. 
I don't know why the `PATH`variable content is different between `pwsh` and `zsh` though. Ahah. Anyway, that's now fixed. Thanks a lot. :)
